docs/UserGuide.md

```diff
@@ -13,151 +27,211 @@ PeopleSoft is a desktop app for **calculating the salary for shift-based contrac
 
 Format: `clear`
 
-Example: `clear` removes all the employees from the app.
+Example: `clear` removes all the employees from the app // ian: does it remove all jobs from the app too?
 
-### Exit the program : `exit`
-Exits the program immediately.
-[coming in v1.3] Requests for confirmation from the user before exiting.
+--------------------------------------------------------------------------------------------------------------------
 
-Format: `exit`
+## Job-related commands
 
-### Save the data
-PeopleSoft saves the data to the hard disk automatically after any command that changes the data. There is no need to save manually.
+### Add a job : `job`
 
-### Edit the data externally
-PeopleSoft data is saved as a JSON file [JAR file location]/data/peoplesoft.json. Advanced users are welcome to update data directly by editing that data file.
+Adds a new job to the system with the given attributes. `RATE` refers to how much the employee is paid per hour. `DURATION` refers to how long the job took.
 
-❗️Caution: If your changes to the data file makes its format invalid, PeopleSoft will discard all data and start with an empty data file at the next run.
+Format: `job JOBID n/NAME r/RATE d/DURATION`
+
+Example: `job 2 n/Fix HDB Lock r/40 d/1` creates a job with id 2, where the employees worked for 1 hour and are paid an hourly rate of $40 to fix a HDB lock
+
+### List all jobs : `joblist`
+
+Lists all the jobs you have created, including jobs that have been paid for and those that have not been paid for yet.
+
+Format: `joblist`
+
+Example: `joblist` lists all the jobs you have created
+
+### Delete a job : `jobdelete`
+
+Deletes the job that was referred to by the index.<br>
+❗️Caution: This is irreversible.
+
+Format: `jobdelete JOBID`
+
+Example: `jobdelete 2` deletes the second job
 
+### Mark a job as paid : `mark`
 
-### Archiving data files `[coming in v2.0]`
+Indicates that a job has been completed.<br>
+❗️Caution: This is irreversible.
 
-_Details coming soon ..._
+Format: `mark JOBID`
+
+Example: `mark 2` marks the second job
+
+### Assign a job to an employee : `assign`
+
+Assigns a job to an employee that is working on it.
+
+Format: `assign JOBID i/PERSON'S_INDEX`
+
+Example: `assign 2 i/3` assigns the second job to the first employee
+
+// To add the generate payslip() function here
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## FAQ
 
+**Q**: How do I save the data?<br>
+**A**: PeopleSoft saves the data to the folder you placed it in automatically after any command that changes the data. There is no need to save manually.
+
+**Q**: What happens if I want to edit the data externally?<br>
+**A**: PeopleSoft data is saved as a JSON file under `/data/peoplesoft.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
i'm not sure we want to "welcome" them to do it, we might just want to say that they "can update the data"

or even better, discourage them from doing so so it's less likely for them to break stuff